package.json

```diff
@@ -41,11 +41,12 @@
     "@angular/router": "~8.2.1",
     "@angular/service-worker": "~8.2.1",
     "@project-sunbird/client-services": "3.9.5",
+    "@project-sunbird/discussions-ui-v8" : "2.0.32-beta.1",
     "@project-sunbird/sb-styles": "0.0.7",
     "@project-sunbird/sunbird-quml-player": "0.0.6",
     "@project-sunbird/telemetry-sdk": "0.0.26",
     "@sunbird-cb/design-system": "0.0.2",
-    "@sunbird-cb/discussions-ui-v8": "^2.0.48",
+    "@sunbird-cb/discussions-ui-v8": "^2.0.50",
```

Comment:
which discussion ui library is used? we need to keep only 1 library right

Reply:
remove the duplicate